x-pack/plugins/observability/public/plugin.ts

```diff
@@ -3,13 +3,37 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import { Plugin as PluginClass, PluginInitializerContext } from 'kibana/public';
+import {
+  AppMountParameters,
+  CoreSetup,
+  DEFAULT_APP_CATEGORIES,
+  Plugin as PluginClass,
+  PluginInitializerContext,
+} from '../../../../src/core/public';
 
 export type ClientSetup = void;
 export type ClientStart = void;
 
-export class Plugin implements PluginClass {
+export class Plugin implements PluginClass<ClientSetup, ClientStart> {
   constructor(context: PluginInitializerContext) {}
-  start() {}
-  setup() {}
+
+  public setup(core: CoreSetup) {
+    core.application.register({
+      id: 'observability-overview',
+      title: 'Overview',
+      order: 7999,
```

Comment:
This seem oddly high, yet specific.

Reply:
Yeah, it looks weird indeed. But that's the way the menu is ordered. I can, of course, change the order of the other Observability plugins in order of this to be the first one (8000). WDYT?

Reply:
It's sorted ascending, so lower order means higher, right?
I'm trying to understand if there is a logic to these numbers. Eg. would be good if observability had the 8xxx namespace. If that's the case then yes, please change this to 8000 and bump the others accordingly.

And perhaps add a comment about this logic.

Reply:
Will do it in that case!